cgiutil.h

```diff
@@ -70,6 +70,12 @@ typedef struct {
   char *postrequest;
 
   char *httpcookiedata;
+
+  // todo, SWIG?
+  const char *path_info;
```

Comment:
Looks like this is breaking the Appveyor/Windows build with `C:\projects\mapserver\mapscript\v8\..\swiginc\..\..\cgiutil.h(75): warning 451: Setting const char * member may leak [C:\projects\mapserver\build\mapscript\python\_pythonmapscript.vcxproj]`

Reply:
It's just a pointer to an environment variable so const is appropriate (I think), but I'll change it and see what happens.

Reply:
No effect... Warning is gone but it was just a warning. Dying with mapscript w/both Travis and Appveyor. I didn't think any of this would have impacted mapscript.

Reply:
Resolved the warning but mapscript isn't building. Odd...

Reply:
Any char **params need a typemap to handle them I think. It looks like the C# bindings are failing on this:

OWSRequest.cs(176,14): error CS0029: Cannot implicitly convert type 'OSGeo.MapServer.SWIGTYPE_p_p_char' to 'string[]' [C:\projects\mapserver\build\mapscript\csharp\mapscript.vcxproj]

Maybe easiest to hide from SWIG for now?